test/unit/data-type.js

```diff
@@ -85,3 +85,56 @@ exports.nanoSecondRounding = function(test) {
   }
   test.done();
 };
+
+exports.dateInputWithSlashSeparator = function(test) {
+  var type = TYPES.typeByName.Date;
+  for (const [value, expectedBuffer] of [
+    ['2015/06/18 23:59:59', Buffer.from('03163a0b', 'hex')],
+  ]) {
+    var buffer = new WritableTrackingBuffer(16);
+    type.writeParameterData(buffer, { value: value }, { useUTC: false });
```

Comment:
Due to the changes introduced via https://github.com/tediousjs/tedious/pull/845, calling `writeParameterData` now requires passing a callback function as the last parameter. I think it'd be enough to pass an empty callback function (like `() => {}`) to fix the test failures here.

Reply:
On more question, why the other test cases called this function does not show this 
symptom？